src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string RuntimeName = ".NET Core";
```

Comment:
@akoeplinger I believe so

@Priya91 @ericstj What does the implementation look like for .NET Framework?